tomoxlending/lendingstate/lendingcontract.go

```diff
@@ -38,7 +38,9 @@ var (
 func IsValidRelayer(statedb *state.StateDB, coinbase common.Address) bool {
 	locRelayerState := GetLocMappingAtKey(coinbase.Hash(), LendingRelayerListSlot)
 
-	if v := statedb.GetState(common.HexToAddress(common.LendingRegistrationSMC), common.BytesToHash(locRelayerState.Bytes())); v != (common.Hash{}) {
+	// a valid relayer must have baseToken
```

Comment:
The reason we need to change this:
https://github.com/tomochain/tomochain/pull/218 allows fee = 0
In that case, `locRelayerState` point to the first element of the struct, meaning point to `fee` 
Therefore, value = common.Hash{} (0000....0000)
To avoid, we check `baseToken` instead of the first element of the struct